content/the-ember-times-issue-187.md

```diff
@@ -1,6 +1,7 @@
 ---
 title: The Ember Times - Issue No. 187
 authors:
+  - anne-greeth-schot-van-herwijnen
```

Comment:
Made https://github.com/ember-learn/ember-blog/pull/1022 to update your name! So exciting BTW!!! 🎉